website/src/main.rs

```diff
@@ -82,17 +84,19 @@ fn switch<G: Html>(route: ReadSignal<Routes>) -> View<G> {
             }
         },
         Routes::Docs(a, b) => {
-            let data = fetch_docs_data(format!("/static/docs/{a}/{b}.json"));
+            let data = fetch_docs_data(format!("/static/docs/{a}/{b}.json",));
```

Comment:
Seems like this was added by accident?
```suggestion
            let data = fetch_docs_data(format!("/static/docs/{a}/{b}.json"));
```